sources/packages/backend/apps/api/src/services/restriction/federal-restriction.service.ts

```diff
@@ -55,25 +55,32 @@ export class FederalRestrictionService extends DataModelService<FederalRestricti
   /**
    * Once the table with all federal restriction is fully imported, these
    * process executes the sequence of bulk operations to:
-   * 1 - Create new active restrictions;
-   * 2 - Deactivate restrictions that are no longer present in the federal data;
-   * 3 - Update the updated_at date for the active restrictions that are still
+   * 1. Create new active restrictions;
```

Comment:
`;`?

Reply:
Even not wrong, as we talked, I changed it because can be easier to accepted 😉